components/ProposalRow.tsx

```diff
@@ -194,8 +195,11 @@ export default function ProposalRow({
           <ProposalState status={status.l1} />
         </div>
         <div className="grow flex flex-col justify-end">
-          <p className="ml-1 mt-1 flex text-xs leading-5 text-gray-500">
-            Created at L1 block {startBlock.l1}
+          <p className="ml-1 flex text-xs leading-5 text-gray-500">
+            L1 Block {createdBlock.toString()}
```

Comment:
I thought the "Created at" gives some useful context, do you think it's not needed?

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/172/commits/60f4a367b6f16140a8e1f36423fbc8020a52dacd